Akiletour/inrage
# Upgrade to NextJS 13 and app directory

## Description
This pull request includes a complete refactor of the application on NextJS 13 with support for the appDirectory. The following changes have been made:

- Refactored the entire application to use NextJS 13, taking advantage of the latest features and improvements.
- Added support for the appDirectory, which allows the application to better handle file-based routing and provides a more scalable structure.

## How to Test
To test these changes, please do the following:

1. Verify that the application still functions as expected after the refactor.
2. Test file-based routing to ensure that it is working correctly.
3. Verify that the application can handle a larger number of pages and components without encountering performance issues.

These updates are designed to improve the performance, scalability, and maintainability of the application by using the latest version of NextJS and taking advantage of the appDirectory.
